examples/BART/BART_introduction.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Is this in the released beta version or the development pymc one? Those warning need to be fixed because otherwise the code will break in 1-2 ArviZ releases.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I run the notebook from main. There are a couple of tweaks in <code>m.bart.plot_dependence</code>  not available in the released beta version.</p>

Reply:
I will then add a label to wait for beta2 to rerun it then merge. Notebooks should run on the env shown in the watermark which would not be the case